Makefile

```diff
@@ -24,6 +24,8 @@ CURRENT_OS=$(shell go env GOOS)
 CURRENT_OS_ARCH=$(shell echo $(CURRENT_OS)-`go env GOARCH`)
 GOBIN?=$(shell echo `go env GOPATH`/bin)
 
+MAIN_GO_MODULE:=github.com/fluxcd/flux
```

Comment:
Bit disappointed to discover that there does not seem to be a Go utility to extract this from the `module` declaration in a `go.mod` file.

Reply:
Uhm, I think it's doable, let me check

Reply:
Done